server/cloudfront-gen-lambda.js

```diff
@@ -150,6 +150,7 @@ function genContentSecurityPolicy() {
     contentSecurityPolicy.imgSrc.push("https://axeptio.imgix.net", "https://www.google.com");
   }
   contentSecurityPolicy.connectSrc.push("https://api.pwnedpasswords.com");
```

Comment:
to be removed if removing password leak code related stuff

Reply:
It will be done when adapting change password with auth0, see [YLP-1524](https://diabeloop.atlassian.net/browse/YLP-1524)